src/theme.jsx

```diff
@@ -53,9 +54,17 @@ const theme = createMuiTheme({
       root: {
         height: 24,
         marginRight: 1,
```

Comment:
Let's change this to `4` to make sure we don't accidentally click on another chip.